src/test/rustdoc-ui/doc-alias-same-name.rs

```diff
@@ -0,0 +1,8 @@
+// compile-flags: -Zdeduplicate-diagnostics=no
+
+#![feature(doc_alias)]
```

Comment:
doc_alias is stable, remember? :)

Reply:
Oh right. Living the unstable way is the only way!